Integrations/MimecastV2/MimecastV2.py

```diff
@@ -421,19 +421,23 @@ def get_policy_request(policy_id=None):
     return response.get('data')
 
 
-def create_policy():
-    headers = ['Policy ID', 'Sender', 'Reciever', 'Bidirectional', 'Start', 'End']
-    contents = {}  # type: Dict[Any, Any]
-    context = {}
-    policies_context = {}  # type: Dict[Any, Any]
-    description = demisto.args().get('description').encode('utf-8')
-    from_part = demisto.args().get('fromPart').encode('utf-8')
-    from_type = demisto.args().get('fromType').encode('utf-8')
-    from_value = demisto.args().get('fromValue').encode('utf-8')
-    to_type = demisto.args().get('toType').encode('utf-8')
-    to_value = demisto.args().get('toValue').encode('utf-8')
-    option = demisto.args().get('option').encode('utf-8')
-
+def get_arguments_for_policy_command(policy_command_type, args):
+    # type: (str, dict) -> (dict, str, Optional[str])
+    """
+      Args:
+          policy_command_type: create / update
```

Comment:
the correct format is:
```suggestion
          policy_command_type (str): create / update
```
And add the args here too